### ISSUE TITLE
DNS configuration to get list of Eureka Servers

### ISSUE THREAD

**Issue by aivans** [SteeltoeOSS/Discovery#8]
Configuration of Eureka Servers through DNS is crucial for having a HA setup.
[https://github.com/Netflix/eureka/wiki/Deploying-Eureka-Servers-in-EC2](url) describes the use of TXT DNS records. The Client would retrieve the list of available Eureka Servers by doing a dns lookup and parsing the list of servers.
`eureka.shouldUseDns=true`

Do you plan to support this?
Otherwise the project is not really usable in a proper production environment.

**Comment by dtillman** [comment]
@aivans : Thanks .. I'll make sure we have a look at this when we add support for AWS environments...

**Comment by aivans** [comment]
@dtillman this is not only AWS. Any proper Eureka deployment should use DNS for HA.

Is it possible to take a closer look at this? I don't think it has much to do with AWS.

**Comment by dtillman** [comment]
@aivans : Yup.. I'll take a closer look ... up till now we have been mostly concerned with the usage of Eureka on Cloud Foundry (CF) and in that setting HA is taken care of easily by scaling the Eureka server to multi instances and the CF platform takes care of the rest...

I'll spend some time looking at this for the non-CF setting.

**Comment by aivans** [comment]
Thank you
